gui/system/forms.py

```diff
@@ -1033,10 +1038,10 @@ def done(self, request, events):
             self.original_instance['stg_guihttpsredirect'] != self.instance.stg_guihttpsredirect or
             self.original_instance['stg_guicertificate_id'] != self.instance.stg_guicertificate_id
         ):
-            if self.instance.stg_guiaddress == "0.0.0.0":
+            if "0.0.0.0" in self.instance.stg_guiaddress:
```

Comment:
`"0.0.0.0" in self.instance.stg_guiaddress.split()` would be more correct because `"0.0.0.0" in "10.0.0.0"` is `True`